en/config_mc/mc_slew_rate_type_trajectory.md

```diff
@@ -78,5 +78,5 @@ These two parameters only have effect during the transition from **velocity-cont
 The purpose of these two parameters are to minimize the jerk introduced from forward flight to hover (please see [MPC_ACC_HOR and MPC_DEC_HOR_SLOW](#mpc_acc_hor-and-mpc_dec_hor_slow)).
 
 The jerk-parameter controls the rate limit with which the acceleration limit can change to `MPC_ACC_HOR_MAX`.
-The actual jerk-value is a linear map from velocity speed to jerk where full speed maps to [MPC_JERK_MAX](../advanced_config/parameter_reference.md#MPC_JERK_MAX) and zero speed to [MPC_JERK_MIN](../advanced_config/parameter_reference.md#MPC_JERK_MIN).
+The actual jerk-value is a linear map from velocity speed to jerk where full speed maps to [MPC_JERK_MAX](../advanced_config/parameter_reference.md#MPC_JERK_MAX) and zero speed to [MPC_JERK_MIN](../advanced_config/parameter_reference.md#MPC_JERK_MIN). 
```

Comment:
@MaEtUgR `MPC_JERK_MIN` appears to have been removed. How should this section (and other mentions in this doc change?)

Reply:
Same as https://github.com/PX4/PX4-user_guide/pull/1305#discussion_r644662752